src/Shortcodes/ImageShortcodeProvider.php

```diff
@@ -69,8 +69,8 @@ public static function handle_shortcode($args, $content, $parser, $shortcode, $e
         // Check if a resize is required
         $src = $record->Link();
         if ($record instanceof Image) {
-            $width = isset($args['width']) ? $args['width'] : null;
-            $height = isset($args['height']) ? $args['height'] : null;
+            $width = (isset($args['width']) && is_numeric($args['width'])) ? $args['width'] : null;
```

Comment:
That would accept some incorrect values, e.g. `+123.45e6`.
Since `ResizedImage` accepts `int` only, shall we rather check for `ctype_digit` and cast to integer afterwards?

E.g.

```php
ctype_digit((string) $args['width']) ? (int) $args['width'] : null;
```

Reply:
Are decimal points acceptable in these attributes? Even if they are it's probably not common for them to be used.

Reply:
Even if they could be, `ResizedImage` method only accepts `int` anyways